doc/client_lua_api.md

```diff
@@ -315,6 +315,7 @@ Call these functions only at load time!
 
 * `minetest.register_globalstep(function(dtime))`
     * Called every client environment step, usually interval of 0.1s
```

Comment:
This is more like 1/FPS or 1/60 IIRC rather than 0.1s, no?

Reply:
It's 0.09 on standalone servers and 1/fps (but at least 1/60) on the client.
This is not something we should document.

Reply:
How about something like "Called every client environment step (every frame, at least 60 times per second)"?

I feel this is worth documenting because accurate calculations may very well rely on the client / server step interval. It's certainly something modders should think about. Serverside, the `dedicated_server_step_setting` should probably be mentioned.

Reply:
If your calculations rely on the number of calls then they're broken.
Neither can we guarantee that it's called 60 times per second, what I meant was "not more than 60 times a second".

Information is good but I don't want to make any promises to modders about this.

Reply:
This is a trivial PR, can we just merge it?

Reply:
Now we can (https://github.com/minetest/minetest/pull/14758/commits/0bebdec715241df2227d3671d5e9a92ced70a393). Client steps being every 0.1s is just misleadingly wrong: It would lead client modders to wrongly believe that this isn't called often enough.

I don't care much about the "usually 0.1s" for server steps statement, this is roughly right considering our default settings at least.